amazon/ion/core.py

```diff
@@ -113,7 +113,19 @@ def scale_to_precision(fractional_seconds, fractional_precision):
     Returns:
         Decimal: The scaled value.
     """
-    return Decimal(fractional_seconds) * Decimal(Decimal(10) ** Decimal(-fractional_precision))
+    return Decimal(fractional_seconds).scaleb(-fractional_precision)
+
+
+def round_to_sixth_precision(fractional_seconds):
+    """Scales the ```fractional_seconds``` attribute to a six digit integer.
```

Comment:
You use the term *round* above and *scale* here, what is really going on?  Its more of a scaling (+ truncation) than it is a rounding, I would make the name and documentation align.

Reply:
Done. Changed method name to `_fractional_seconds_to_microseconds`.